bioimageio/core/prediction.py

```diff
@@ -342,7 +362,8 @@ def predict_with_tiling(
             if any(sc != 1 for ax, sc in scale.items() if ax in "xyz") or any(
                 off != 0 for ax, off in offset.items() if ax in "xyz"
             ):
-                raise NotImplementedError("Tiling with a different output shape is not yet supported")
+                pass
```

Comment:
I think we can just remove this whole check, including the `if`.

Reply:
Done!